vta/apps/tsim_example/hardware/chisel/.scalafmt.conf

```diff
@@ -0,0 +1,2 @@
+maxColumn = 80
```

Comment:
I would suggest setting this number to `100`, see `max-line-length` definition in the current linter [here](https://github.com/dmlc/tvm/blob/master/tests/lint/pylintrc#L101).

Reply:
I just changed to 100